src/zoo.js

```diff
@@ -10,71 +10,105 @@ eslint no-unused-vars: [
 */
 
 const data = require('./data');
+const { sortArray, getAnimals } = require('./sort');
```

Comment:
Aqui não seria assim?
```suggestion
const { sortArray, getAnimals } = require('./handlers');
```